source/Files.cpp

```diff
@@ -545,10 +546,9 @@ void Files::Write(FILE *file, const string &data)
 
 
 
-void Files::LogError(const string &message)
+void Files::LogErrorToFile(const string &message)
```

Comment:
Should this be moved into Logger?

Reply:
No, I feel it is much better to keep it here, because the Files class has a dependency on SDL for determining the paths on where to store things, and the main goal of this PR is to remove the SDL dependencies out of the main Logging code.